con-master/src/main/resources/application.properties

```diff
@@ -1,9 +1,12 @@
 spring.datasource.url=jdbc:mysql://localhost/concierge
 spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.password=1234
 spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+        
+cors.allowed.origins=http://localhost:3000
+
+image.directory.path=${user.home}/var/concierge/images/
```

Comment:
`var` should probably be removed. `var` is a well known directory in root of Unix based systems. `concierge/images/` should be enough.